### PR TITLE
Update Observability docs for Cloud 2.0 - update Metrics Overview

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9980

In metrics.md, (a) added Available on SQL tab for Standard, (b) corrected capitalization of CloudWatch.

### CODE REVIEW

File: src/current/cockroachcloud/metrics.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The time interval selector at the top of each tab allows you to filter the view
 
 CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.advanced }} users can export metrics to the following third-party cloud sinks:
 
-- AWS Cloudwatch
+- AWS CloudWatch
```
Comment (taroface) [inline src/current/cockroachcloud/metrics.md:43]: I think this is technically `Amazon CloudWatch`.

File: src/current/cockroachcloud/metrics.md [hunk 2]
```diff
@@ -28,8 +28,8 @@ Depending on your CockroachDB {{ site.data.products.cloud }} deployment, the **M
 | [SQL]({% link cockroachcloud/metrics-sql.md %}) | ✔ |  |  |
 | [Changefeeds]({% link cockroachcloud/metrics-changefeeds.md %}) | ✔ |  |  |
 | [Row-Level TTL]({% link cockroachcloud/metrics-row-level-ttl.md %}) | ✔ |  |  |
-| [Monitor SQL Activity]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) |  | ✔ | ✔ |
-| [Identify SQL Problems]({% link cockroachcloud/metrics-identify-sql-problems.md %}) |  | ✔ | ✔ |
+| [Monitor SQL Activity]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) | available on [SQL]({% link cockroachcloud/metrics-sql.md %}) tab | ✔ | ✔ |
```
Comment (taroface) [inline src/current/cockroachcloud/metrics.md:31]: "Available" should be capitalized.

But also, I'm confused what this means. Is this still accessible through the Metrics page? If so, I think a checkmark might still be clearest.

File: src/current/cockroachcloud/metrics.md [hunk 3]
```diff
@@ -28,8 +28,8 @@ Depending on your CockroachDB {{ site.data.products.cloud }} deployment, the **M
...
+| [Identify SQL Problems]({% link cockroachcloud/metrics-identify-sql-problems.md %}) | available on [SQL]({% link cockroachcloud/metrics-sql.md %}) tab | ✔ | ✔ |
```
Comment (taroface) [inline src/current/cockroachcloud/metrics.md:32]: Ditto

File: src/current/cockroachcloud/metrics.md (hunk 2 above)
Comment (florence-crl) [reply src/current/cockroachcloud/metrics.md]: updated with "Same graphs are available on Metrics page SQL tab"

File: src/current/cockroachcloud/metrics.md (hunk 3 above)
Comment (florence-crl) [reply src/current/cockroachcloud/metrics.md]: updated with "Same graphs are available on Metrics page SQL tab"

File: src/current/cockroachcloud/metrics.md (hunk 1 above)
Comment (florence-crl) [reply src/current/cockroachcloud/metrics.md]: Thanks for the correction! I updated it to Amazon CloudWatch. I also created [DOC-10018](https://cockroachlabs.atlassian.net/browse/DOC-10018) to correct this throughout the repo.

**Review by taroface — APPROVED** [review]
LGTM but a suggested edit on the SQL tab columns - sorry for the back and forth.

File: src/current/cockroachcloud/metrics.md [hunk 4]
```diff
@@ -23,14 +23,14 @@ Depending on your CockroachDB {{ site.data.products.cloud }} deployment, the **M
...
-| [Row-Level TTL]({% link cockroachcloud/metrics-row-level-ttl.md %}) | ✔ |  |  |
-| [Monitor SQL Activity]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) |  | ✔ | ✔ |
-| [Identify SQL Problems]({% link cockroachcloud/metrics-identify-sql-problems.md %}) |  | ✔ | ✔ |
-| [Custom]({% link cockroachcloud/custom-metrics-chart-page.md %}) | ✔ | ✔ | ✔ |
+| [**Overview**]({% link cockroachcloud/metrics-overview.md %}) | ✔ |  |  |
+| [**Request Units**]({% link cockroachcloud/metrics-request-units.md %}) | ✔ |  |  |
+| [**SQL**]({% link cockroachcloud/metrics-sql.md %}) | ✔ |  |  |
+| [**Changefeeds**]({% link cockroachcloud/metrics-changefeeds.md %}) | ✔ |  |  |
+| [**Row-Level TTL**]({% link cockroachcloud/metrics-row-level-ttl.md %}) | ✔ |  |  |
+| [**Monitor SQL Activity**]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) | Same graphs are available<br>on **Metrics** page [**SQL**]({% link cockroachcloud/metrics-sql.md %}) tab | ✔ | ✔ |
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/cockroachcloud/metrics.md]: updated

**Pull request merged**